Dockerfile.centos

```diff
@@ -186,6 +186,9 @@ RUN make install
 
 RUN cp -v /etc/ssl/certs/ca-bundle.crt /etc/ssl/certs/ca-certificates.crt
 
+WORKDIR /example_cpp_client
```

Comment:
First we probably don't need WORKDIR here,
second why copy client into OVMS image? Shouldn't copying it into static analysis scan image suffice?

Reply:
Static analysis scan does not have ovms repository context. The only thing we have there is base image which is OVMS-build image. This is why c++ client must end up there.